packages/components/statistic/src/index.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div :class="[ns.b()]">
+    <div class="head">
+      <slot name="title">
+        <span class="title">
+          {{ title }}
+        </span>
+      </slot>
+    </div>
+    <div class="con">
+      <span class="prefix">
```

Comment:
Need prefix here.

Reply:
`:class="ns.e('prefix')" ` Is that it? Or is it something else?